cockatrice/src/dlg_forgotpasswordreset.cpp

```diff
@@ -0,0 +1,101 @@
+#include <QLabel>
+#include <QGridLayout>
+#include <QHBoxLayout>
+#include <QDialogButtonBox>
+#include <QMessageBox>
+#include <QDebug>
+
+#include "dlg_forgotpasswordreset.h"
+
+DlgForgotPasswordReset::DlgForgotPasswordReset(QString host, int port, QString playerName, QWidget *parent)
+    : QDialog(parent)
+{
+    hostLabel = new QLabel(tr("Host:"));
+    hostEdit = new QLineEdit(host);
+	hostLabel->hide();
```

Comment:
I understand having `hostEdit` but I don't get why we need the `hostLabel`. That's what i'm asking